.cargo/config.toml

```diff
@@ -1,24 +1,17 @@
 [target.xtensa-esp32-espidf]
 linker = "ldproxy"
 runner = "espflash --monitor"
-rustflags = ["-C", "default-linker-libraries", "-Clink-args=-Wl,-Map=./esp-build.map", "--cfg", "espidf_time32"]
-
+rustflags = ["--cfg", "espidf_time32"]
```

Comment:
`"-C", "default-linker-libraries"` was never intended for application on xtensa targets, it was only to work around https://github.com/esp-rs/embuild/issues/16 on riscv, which we don't currently support. If and when we *do* support it, the upstream issue may be resolved and we won't need it.

Also, the esp-build.map thing doesn't seem to get produced, at least for me. I'm going to double check that in a few different environments before committing to removing it, but if I can't get the file to be emitted I don't think we should have the flag.

Reply:
doesn't get produced for me either